public/javascripts/index.js

```diff
@@ -6,52 +6,59 @@ function scrollToElement (element) {
 }
 
 function renderError (el, message) {
-  console.log('Appending: ' + message)
-  el.text(' - ' + message)
+  console.log(message)
```

Comment:
winston is only available on the server, this in on client. I should be able to just get rid of this line actually, b/c the same message shows up on the page somewhere